tests/sync/test_settings.py

```diff
@@ -0,0 +1,36 @@
+import pathlib
+
+from kitovu.sync.settings import Settings, PluginSettings
+
+
+def test_load_a_sample_yaml_file():
+    settings = Settings.from_yaml_file(pathlib.PurePath('./tests/assets/smb_example_config.yml'))
+
+    expected_root_dir = pathlib.PurePath('~/Documents/HSR/semester_06')
+    expected_global_ignore = ['Thumbs.db', '.DS_Store']
+    expected_plugins = {
+        'skripte-server': PluginSettings(
+            plugin_type='smb',
+            connection={
+                'username': 'example_user',
+            },
+            syncs=[
+                {
+                    'name': 'Engineering-Projekt',
+                    'ignored': ['SubDir', 'example.txt'],
+                    'remote-dir': pathlib.PurePath('Informatik/Fachbereich/Engineering-Projekt/EPJ'),
+                    'local-dir': pathlib.PurePath('~/Documents/HSR/semester_06/Engineering-Projekt'),
+                }
+            ],
+        ),
+    }
+
+    assert settings.root_dir == expected_root_dir
+    assert settings.global_ignore == expected_global_ignore
+    assert settings.plugins == expected_plugins
+
+    assert settings == Settings(
```

Comment:
This seems to test (almost) the same thing as the three lines above.

Reply:
If anything is different with just the assert of the settings the output is not clear (at least for me).
Thats why I've added the first three asserts, but kept the asserts of the settings if some properties would be added or the constructor would change values or something like that